contrib/scalar/scalar.c

```diff
@@ -158,12 +133,10 @@ static int set_recommended_config(void)
 		{ "receive.autoGC", "false" },
 		{ "reset.quiet", "true" },
 		{ "status.aheadBehind", "false" },
-#ifdef WIN32
-		/*
-		 * Windows-specific settings.
-		 */
+#ifndef WIN32
 		{ "core.untrackedCache", "true" },
-		{ "core.filemode", "true" },
+#else
+		{ "core.untrackedCache", "false" },
 #endif
```

Comment:
nit: would it not be more "natural" to invert this `#ifndef` to an `#ifdef`?

"if windows then X else Y"

Reply:
In this instance, I thought that we'd use the desirable arm first: "_unless_ this is Windows, we use the untracked cache".

I should probably add a comment to clarify that this is unfortunate and that we'll want to spend some quality time trying to make untracked cache work on Windows and get rid of the `#else`.

Reply:
Did that: https://github.com/microsoft/git/pull/350/commits/94d7b05b4c8a187f17358d3d076a49846d61cbcd#diff-f9053a8d79e6d409ddf144989d8202388ac5edadbdc3b6140a6b94c894110950R140-R149